### DISCUSSION TITLE
Mixed results search method

### DISCUSSION THREAD

**Discussion by ja7ad** [meilisearch/product#771] [Feedback & Feature Proposal]
We have 3 methods to search:

- normal search
- multi search 
- facet search

I think we need to a new search method to get mixed results from multiple indexes.

For example:

```shell
POST /mixed-search
```

**Body:**

- query as string 
- array of indexes object
     - uid as string
     - attributes as array of string
     - limit
    - offset
    - filter
    - sort
    - ...

**Response:**

Results of mixed attributes selected in the index object body with value .

**Comment by ManyTheFish** [comment]
Hello @Ja7ad,

In the next release (v1.10), you may be able to do it by using the federated search feature. You can already try it with the pre-release:
https://github.com/meilisearch/meilisearch/releases/tag/v1.10.0-rc.0

Let us know if this feature fulfills your needs!
